src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/db/Document.kt

```diff
@@ -36,7 +36,11 @@ data class Document(
   val createdByUserId: UUID,
 
   @Column(nullable = false)
-  val createdDateTime: OffsetDateTime
+  val createdDateTime: OffsetDateTime,
+
+  @Column(nullable = true)
+  val details: String?
+
```

Comment:
I have tended to leave the createdBy fields last. Can you move this above `createdByUserId` and rearrange the constructor ordering. sorry 😬